File: samples/wordnik-api-scala/src/main/scala/com/wordnik/client/model/WordSearchResults.scala
package com.wordnik.client.model

import scala.reflect.BeanProperty

class WordSearchResults {
  @BeanProperty var totalResults: Int = 0
  @BeanProperty var searchResults: java.util.List[WordSearchResult] = new java.util.ArrayList[WordSearchResult]()
  override def toString: String = {
    val sb = new StringBuilder
    sb.append("class WordSearchResults {\n")
    sb.append("  totalResults: ").append(totalResults).append("\n")
    sb.append("  searchResults: ").append(searchResults).append("\n")
    sb.append("}\n")
    sb.toString
  }
}

File: samples/wordnik-api-java/src/main/java/com/wordnik/client/model/WordSearchResults.java
package com.wordnik.client.model;

import java.util.*;
public class WordSearchResults {
  private int totalResults = 0;
  private List<WordSearchResult> searchResults = new ArrayList<WordSearchResult>();
  public int getTotalResults() {
    return totalResults;
  }
  public void setTotalResults(int totalResults) {
    this.totalResults = totalResults;
  }

  public List<WordSearchResult> getSearchResults() {
    return searchResults;
  }
  public void setSearchResults(List<WordSearchResult> searchResults) {
    this.searchResults = searchResults;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class WordSearchResults {\n");
    sb.append("  totalResults: ").append(totalResults).append("\n");
    sb.append("  searchResults: ").append(searchResults).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
